File: test/fixtures/Sample.java
package com.example;

import java.util.List;
import java.util.Map;

/** InventoryService manages stock. */
public class InventoryService implements Reader {
    private String db;
    public static final int MAX = 100;

    public InventoryService(String db) {
        this.db = db;
    }

    public boolean reserve(String sku, int qty) {
        return qty > 0;
    }

    private void helper() {}
}

interface Reader {
    int read();
    void close();
}

enum Color {
    RED,
    GREEN
}

File: test/fixtures/Sample.cs
using System;
using System.Collections.Generic;

namespace App
{
    /// <summary>InventoryService manages stock.</summary>
    public class InventoryService : IReader
    {
        private string db;
        public int Count { get; set; }

        public InventoryService(string db)
        {
            this.db = db;
        }

        public bool Reserve(string sku, int qty) => qty > 0;

        private void Helper() {}
    }

    public interface IReader
    {
        int Read();
    }

    public enum Color
    {
        Red,
        Green
    }

    public struct Point
    {
        public int X;
    }
}
